lib/util/xml_utils.js

```diff
@@ -402,3 +420,23 @@ shaka.util.XmlUtils = class {
     }
   }
 };
+
+/**
+ * Promote a string to TrustedHTML. This function is security-sensitive and
+ * should only be used with security approval where the string is guaranteed not
+ * to cause an XSS vulnerability.
+ *
+ * @private {!shaka.util.Lazy.<function(!string): (!TrustedHTML|!string)>} */
```

Comment:
Style nit: The `*/` should be on the next line.

Reply:
Good catch, fixed.